.github/workflows/cd-event.yml

```diff
@@ -21,20 +21,31 @@ jobs:
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v4.1.1
+        uses: actions/checkout@v4.1.6
 
       - uses: hei-school/aws-credentials-setter@v1.0.3
         with:
           secrets: ${{ toJSON(secrets) }}
           region: ${{ env.AWS_REGION }}
 
-      - name: Deploy event stack
+      - name: Deploy primary event stack
         uses: aws-actions/aws-cloudformation-github-deploy@v1
         with:
-          name: ${{ github.ref_name }}-event-poja-base
+          name: ${{ github.ref_name }}-primary-event-poja-base
```

Comment:
rather: use queue-1

Thus if we want to generalize to n queues later on, we would have no backward comp pb.